diff --git a/decoupled_router.module b/decoupled_router.module
index 8ad80c6..cdab351 100755
--- a/decoupled_router.module
+++ b/decoupled_router.module
@@ -8,11 +8,13 @@
 use Drupal\path_alias\PathAliasInterface;
 
 /**
- * Implements hook_path_update().
+ * Implements hook_ENTITY_TYPE_update().
+ *
+ * @param PathAliasInterface $path_alias
  */
-function decoupled_router_path_update($path) {
+function decoupled_router_path_alias_update(PathAliasInterface $path_alias) {
   \Drupal::service('decoupled_router.cache_invalidation')
-    ->invalidateByPath($path);
+    ->invalidateByPath($path_alias->getPath());
 }
 
 /**
@@ -25,9 +27,11 @@ function decoupled_router_path_alias_insert(PathAliasInterface $path_alias) {
 }
 
 /**
- * Implements hook_path_delete().
+ * Implements hook_ENTITY_TYPE_delete().
+ *
+ * @param PathAliasInterface $path_alias
  */
-function decoupled_router_path_delete($path) {
+function decoupled_router_path_alias_delete(PathAliasInterface $path_alias) {
   \Drupal::service('decoupled_router.cache_invalidation')
-    ->invalidateByPath($path);
+    ->invalidateByPath($path_alias->getPath());
 }
